docs/sphinx/examples/cpp/other/trotter_kernel_mode.cpp

```diff
@@ -121,23 +122,31 @@ int run_steps(int steps, int spins) {
     auto words = term_words(ham);
     auto magnetization_exp_val = cudaq::observe(
         trotter{}, average_magnetization, &state, coefficients, words, dt);
-    expResults.emplace_back(magnetization_exp_val.expectation());
+    auto result = magnetization_exp_val.expectation();
+    expResults.emplace_back(result);
     state = cudaq::get_state(trotter{}, &state, coefficients, words, dt);
     const auto stop = std::chrono::high_resolution_clock::now();
     auto duration =
         std::chrono::duration_cast<std::chrono::microseconds>(stop - start);
-    runtimeMs.emplace_back(duration.count() / 1000.0);
+    auto timeInSeconds = duration.count() / 1000.0 / 1000.0;
+    runtimeMs.emplace_back(timeInSeconds);
+    std::cout << "Step " << i << ": time [s]: " << timeInSeconds
+              << ", result: " << result << std::endl;
```

Comment:
Just a note: we typically favor `'\n'` instead of `std::endl` because `std::endl` forces a flush, and that flush has a throughput cost. However, in this example case, it's probably _better_ to force the flush to happen so the user is guaranteed to see the output as it happens. That's a long way of saying: you don't need to change anything here 😄

Reply:
Got it, thanks!